### DISCUSSION TITLE
ABSTRCT 字体样式修改；插入签名图片；英文扉页的 Author 和 Supervisor 对齐问题；修改 arxiv 参考文献的格式

### DISCUSSION THREAD

**Discussion by CrazyElements** [sjtug/SJTUThesis#1035] [提问]
1. 英文摘要 "abstract" 字体样式与学院要求的不符
![image](https://github.com/sjtug/SJTUThesis/assets/105211051/16aad5fd-43f6-4351-8396-fbd5528f28a4)
2. 学位论文原创性声明签名处插入签名图片(如 png)：我知道可以在 PDF 中插入，但我想知道是否能在 `sjtu-name-thesis-zh.def` 中插入图片
3. 英文扉页的 Author 行和 Supervisor 行要求冒号对齐
![image](https://github.com/sjtug/SJTUThesis/assets/105211051/5a07a0b3-bf97-441b-b191-422eabb51b35)
4. 学院要求 arxiv 文章的参考格式如下，需要创建日期和访问日期等
![image](https://github.com/sjtug/SJTUThesis/assets/105211051/304fcb74-ff29-4df0-89cf-f09e091c4c68)

谢谢大家

**Comment by LogCreative** [comment]
1. 字体已经是 Arial，应该是字重上有区别，但是理论上应当是粗体与中文摘要名的字重情况保持一致。

2. 目前的代码结构不适宜直接插入图片，而且归档时一般要求手写签名，如确有需要建议直接插入PDF复印件，或者通过PDF编辑软件手动插入。

3. 属于细微差别，不是关键问题。该问题已经被 SJTUTeX v2.1 通过表格重构解决。

4. 见 #971

**Reply by CrazyElements** [reply]
另外，请问 3 如何在 sjtuthesis 中修改啊？我看了 sjtutex.dtx 中关于英文扉页的内容，想仿照这修改，但还是不会，都快疯了:(
![image](https://github.com/sjtug/SJTUThesis/assets/105211051/61d6262b-643b-4e87-ba32-deca162435f4)

还有就是原创性声明的文字部分，似乎也和学校要求的不一样呢，这要怎么改啊（字体大小都一样了，似乎是页边距的问题？）
![image](https://github.com/sjtug/SJTUThesis/assets/105211051/f03bf034-a1a3-418b-a84e-181c2d0691ea)
![image](https://github.com/sjtug/SJTUThesis/assets/105211051/3b13f738-3ae7-4fff-b526-5ace68cf552e)
学校今年的 arxiv 引用规则需要去除"预印本平台:预印本编号", 即今年的格式是
> 主要责任者.题名:其他题名信息[EB/OL].(更新或修改日期)[引用日期].获取和访问路径

请问在 4 的基础上，需要作出哪些修改才能删除这部分的信息呢？

**Reply by LogCreative** [reply]
2. 原创性声明学校 Word 模板的格式跟系统里导出的格式也是不同的，这个区别不重要。
4. 如果使用的是我的方法，直接删去对应bib条目的
```
      eprint={1808.01340},
      archivePrefix={arXiv},
```

**Reply by CrazyElements** [reply]
好的，谢谢您，我在删去这两行，再添加 urldate 和 url 项后，得到了和学校对 arxiv 引用要求相同的结果。

**Reply by 2212582443** [reply]
你好，我在v2-dev分支代码中使用了你提到的方法更改arxiv引用，将bib文件中@article（之前引用的时候直接复制google scholar搜索结果的引用代码）替换为@misc（是arxiv网站提供的bib文件），但是替换后原位置参考文献直接无法显示，也就是参考文献数量减一，请问这可能是什么问题，应当如何解决？

**Reply by LogCreative** [reply]
`@article{key1, ....}` 和 `@misc{key2, ...}` 里面的主键可能不一样，检查一下是 `\cite{key2}` 而不是 `\cite{key1}`。

**Comment by AlexaraWu** [comment]
> 1. 字体已经是 Arial，应该是字重上有区别，但是理论上应当是粗体与中文摘要名的字重情况保持一致。

Word 模板其实没用对字体，它这里实际上用的是 Arial Unicode MS 伪粗体，和正常的 Arial 粗体有区别。
